install_config/install/deploy_router.adoc

```diff
@@ -1113,74 +1113,87 @@ endif::[]
 
 Save this content to a file for use as the basis of your customized template.
 
-[[using-configmap-replace-template]]
 
+[[using-configmap-replace-template]]
 === Using a ConfigMap to Replace the Router Configuration Template
 
 A xref:../../dev_guide/configmaps.adoc#[configMap] permits you to customize
-the router instance without rebuilding the router image. The
-*_haproxy-config.template_*, *_reload-haproxy_* and other scripts can be modified
-as well as creating and modifying router environment variables.
+the router instance without rebuilding the router image.
+You can modify the *_haproxy-config.template_*, *_reload-haproxy_*, and
+other scripts, as well as create and modify router environment variables.
 
-The following example shows how to use a custom *_haproxy-config.template_* in
-your router.
+The following example shows how to use
+a custom *_haproxy-config.template_* file.
 
-First get a copy of the *_haproxy-config.template_* that you want to modify as
-xref:#deploying-customized-router[described above]. Modify the
-*_haproxy-config.template_* as desired.
+. Get a copy of the *_haproxy-config.template_* that you want to modify as
```

Comment:
"Get a copy" sounds...dodgy. Should it be something like:

"Copy the <file> template that you want to modify..."

?